File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int i=0;
        int n=nums.length;
        int j=1;
        int count=1;
        while(j<n){
            if(nums[i]== nums[j]){
                j++;
            }
            else{
                i++;
                nums[i]=nums[j];
                j++;
                count++;
            }
        }
        return count;
    }
}

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        n=len(nums)
        L=1
        R=0

        while R<n-1:
            if(nums[R]==nums[R+1]):
                R+=1
            else:
                nums[L]=nums[R+1]
                L+=1
                R+=1
        return L